lib/pymedphys/tests/dicom/test_connect.py

```diff
@@ -250,7 +268,6 @@ def test_dicom_listener_cli(test_dataset):
             assert status.Status == 0
             assoc.release()
 
-        series_dir = test_directory.joinpath(test_dataset.SeriesInstanceUID)
-        file_path = series_dir.joinpath(f"RP.{test_dataset.SOPInstanceUID}.dcm")
+        file_path = _build_hierarchical_path_to_plan(test_directory, test_dataset)
```

Comment:
Nit.

Move this function into the library and utilise it within the class (as discussed above)? Then, the `hierarchical_dicom_storage_directory` function can be tested in a standalone fashion with something like:


```python
patient_id = '000'
uid_a = pydicom.uid.generate_uid()
uid_b = pydicom.uid.generate_uid()

dataset = dicom_dataset_from_dict({
	'PatientID': patient_id ,
	'StudyInstanceUID': uid_a,
    'SeriesInstanceUID': uid_b
})

root_dir = pathlib.Path('a/directory')
expected_directory = root_dir.joinpath(patient_id, uid_a, uid_b)

created_directory = hierarchical_dicom_storage_directory(root_dir, dataset)
assert created_directory == expected_directory
```